### DISCUSSION TITLE
Minimal Compile Options for OBJ Loading

### DISCUSSION THREAD

**Discussion by cuppajoeman** [assimp/assimp#5477] [Q&A]
Hey there, 

I'm using assimp to load in obj's for my game, as of right now I just want it to load objs and I don't care about any other feature, I have this in my cmake:
```
set(ASSIMP_INSTALL OFF CACHE BOOL "" FORCE)
set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
set(ASSIMP_BUILD_TESTS OFF CACHE BOOL "" FORCE)
set(ASSIMP_NO_EXPORT OFF CACHE BOOL "" FORCE)
set(ASSIMP_BUILD_ALL_EXPORTERS_BY_DEFAULT OFF CACHE BOOL "" FORCE)
set(ASSIMP_BUILD_ALL_IMPORTERS_BY_DEFAULT OFF CACHE BOOL "" FORCE)
set(ASSIMP_BUILD_OBJ_IMPORTER ON CACHE BOOL "" FORCE)
```
But when I compile I still see a lot of stuff about postprocessing and other things.

Can someone provide me with the compile options that will literally only give me obj loading (if possible). 

Thanks

**Comment by kimkulling** [comment]
The postprocessing steps are mandatory. I guess to make your build less big use the release build. You can just not use any kind of prost processing and that is all.
